thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+          "tooltip": "Specify the message of the observable you want to update for alert in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Add Tags",
+          "name": "addTags",
+          "type": "text",
+          "description": "Specify CSV list of tags will be added to the current observable.",
+          "tooltip": "Specify CSV list of tags will be added to the current observable.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Remove Tags",
+          "name": "removeTags",
+          "type": "text",
+          "description": "Specify CSV list tags will be removed from the current observable",
+          "tooltip": "Specify CSV list of tags will be removed from the current observable",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "TLP",
+          "name": "ignoreSimilarity",
+          "type": "checkbox",
+          "description": "Specify the TLP of the observable you want to update.",
+          "tooltip": "Specify the TLP of the observable you want to update.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Additional Field",
+          "name": "additional_field",
+          "type": "json",
+          "description": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "tooltip": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        }
+      ],
+      "enabled": ""
+    },
+    {
+      "title": "Delete Observable",
+      "operation": "delete_observable",
+      "description": "Delete a specific observable from the TheHive platform based on the input parameters you have specified.",
+      "category": "investigation",
+      "annotation": "delete_observable",
+      "output_schema": {
+        "type": "",
+        "message": ""
+      },
+      "parameters": [
+        {
+          "title": "Observable ID",
+          "name": "observableId",
+          "type": "text",
+          "description": "Specify the ID of the observable from which you want to retrieve information from TheHive Platform.",
+          "tooltip": "Specify the ID of the observable from which you want to retrieve information from TheHive Platform.",
+          "visible": true,
+          "editable": true,
+          "required": true
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "title": "Create Task in Case",
+      "operation": "create_task",
+      "description": "Creates a task in the in TheHive based on the input parameters you have specified.",
```

Comment:
Please correct the description for this line: "in the in TheHive". Make necessary changes in all operations if required.